platform/qemu.go

```diff
@@ -364,3 +377,98 @@ func Virtio(board, device, args string) string {
 	}
 	return fmt.Sprintf("virtio-%s-%s,%s", device, suffix, args)
 }
+
+func setupIgnition(confPath string, diskImagePath string) error {
+	fileRemoteLocation := "/boot/ignition/config.ign"
```

Comment:
nit: (does not need to be updated, maybe a future change) Seems like this should be a constant that is referenced.